lib/include/ert/util/test_work_area.hpp

```diff
@@ -38,15 +79,12 @@ extern "C" {
   void                  test_work_area_free(test_work_area_type * work_area);
   const char          * test_work_area_get_cwd( const test_work_area_type * work_area );
   const char          * test_work_area_get_original_cwd( const test_work_area_type * work_area );
-  void                  test_work_area_install_file( test_work_area_type * work_area , const char * input_src_file );
```

Comment:
Isn't this function conflicting a bit with the idea of a `tmp` test area? It does not seems to be much used either. What would you think about keeping it out of the C++ API?

Reply:
The purpose of this functionality is to copy a file - with leading directory components, from somewhere down to the testarea. It is used in a couple of tests in libres; would prefer to leave it - at least until the tests in libres can be modified.